src/networking/backend.ts

```diff
@@ -169,6 +177,11 @@ export class Backend {
         presentedOfferingContext.placementIdentifier;
     }
 
+    if (this.purchasesContext?.workflowIdentifier) {
+      requestBody.presented_workflow_id =
+        this.purchasesContext.workflowIdentifier;
```

Comment:
We might end up adding an intermediate `PresentedWorkflowContext` in the base PR, so this might change slightly, but can wait until that's done :)